### PR TITLE
Update readme and vignette

### PR DESCRIPTION
**Pull request by torfason**
Various updates to readme and vignette. This PR is probably not ready to merge, but can be informally reviewed and commented on. It is probably best to merge #44 before this one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,182 +1,176 @@
...
-p <- qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+```R
+import::from(magrittr, "%>%", "%$%", .into = "operators") 
+import::from(dplyr, arrange, .into = "datatools")
+```
 
-## A function relying on a function exposed through attachment:
-plot_it <- function()
-  qplot(Sepal.Length, Sepal.Width, data = iris, color = Species)
+If using pied-à-terres actively, one might prefer the alternative syntax
```
Comment (smbache) [inline README.md:134]: I think this word, and reasoning, was removed above. Maybe use a more standard word, or reintroduce it.

File: README.md [hunk 2]
```diff
@@ -1,182 +1,176 @@
...
-b
-p
+# Will not work (import will look for a package called "module_name")
+import::from(module_name, foo, bar)
 
-## Does not work, as ggplot2 is no longer attached
-## (would work with the import statement!):
-#plot_it()
+# This will correctly import the foo() and bar() functions from "../utils/my_module.R"
+import::from(module_name, foo, bar)
```
Comment (smbache) [inline README.md:160]: is this not the same as above?

File: README.md (hunk 1 above)
Comment (torfason) [reply README.md:134]: Added a short intro before this usage.

File: README.md (hunk 2 above)
Comment (torfason) [reply README.md:160]: Added `.character_only=TRUE` to the second instance, which was supposed to be the difference.

**Comment by torfason** [comment]
After the last revisions, I'd like to propose this as a release version. Of course one can always improve things (and we should improve any specific issues that are found), but I think this version brings enough benefit to justify a release. In terms of the docs changes, we now have:

  * A `README.md` that provides brief examples and intro
  * A vignette that should be pretty complete
  * A straightforward way to navigate between different parts of the documentation
  * No undocumented features that I am aware of.

**Comment by torfason** [comment]
Spelling issues have now been fixed, and version has been upgraded to 1.2.0

**Review by smbache — APPROVED** [review]

**Pull request merged**